opossum-tool/OpossumUI
# chore: improve unit test cleanup

### Summary of changes

- create test output directory in global setup
- remove test output directory in global teardown
- make sure unit tests write artifacts into output folder by providing output path via new faker method
- convert jest config to TypeScript

### Context and reason for change

fixes issue #2078

### How can the changes be tested

Run `yarn test:unit` in terminal and note that the test folder, "test-output" should be created and then deleted again at the end of the run. Should also work on Windows.
